applications/crossbar/src/modules/cb_tasks.erl

```diff
@@ -284,6 +290,13 @@ put(Context) ->
             crossbar_util:response_400(<<"bad request">>, Reason, Context)
     end.
 
+-spec task_account_id(cb_context:context()) -> api_ne_binary().
+task_account_id(Context) ->
+    case cb_context:account_id(Context) of
+        'undefined' -> cb_context:auth_account_id(Context);
```

Comment:
I wanted to hit /v2/tasks as super duper admin to upload a ratedeck, which isn't tied to an account